Card and CardGrid :) (#2007)
liferay/clay: next.clayui.com/content/docs/migration/migrate-the-clay-components-from-v2-to-v3.md

```diff
- <ClayUserCard name="User Name" spritemap={spritemap} selectable />
+ <ClayCard displayType="user" selectable>
+ 		<ClayCard.Header>
+ 			<ClayCheckboxWithState>
+ 				<ClayCard.AspectRatio>
+ 					<div className="aspect-ratio-item aspect-ratio-item-center-middle aspect-ratio-item-fluid card-type-asset-icon">
+ 						<ClaySticker
+ 							className="sticker-user-icon"
+ 							displayType="secondary"
+ 							shape="circle"
+ 						>
+ 							<ClayIcon
+ 								spritemap={props.spritemap}
+ 								symbol="user"
+ 							/>
+ 						</ClaySticker>
+ 					</div>
+ 				</ClayCard.AspectRatio>
+ 			</ClayCheckboxWithState>
+ 		</ClayCard.Header>
+ 
+ 		<ClayCard.Body>
+ 			<div className="autofit-col autofit-col-expand">
+ 				<ClayCard.Description type="title">
+ 					Adélaide
+ 				</ClayCard.Description>
+ 				<ClayCard.Description type="subtitle">
+ 					Author Action
+ 				</ClayCard.Description>
+ 				<ClayCard.Detail>
+ 					<ClayLabel displayType="warning">{'Rejected'}</ClayLabel>
+ 				</ClayCard.Detail>
+ 			</div>
+ 		</ClayCard.Body>
+ 	</ClayCard>
```

### API Changes:

> - Only the `spritemap` property was keeped comparing to the v2 API. Take a look on the documentation if you have any doubts.

Reviewer: ```suggestion
- Only the `spritemap` property was kept compared to the v2 API.
```

+ 				DOC
+ 			</ClaySticker>
+ 		</ClayCheckboxWithState>
+ 	</ClayCard.Header>
+ 	<ClayCard.Body>
+ 		<div className="autofit-col autofit-col-expand">
+ 			<section className="autofit-section">
+ 				<ClayCard.Description type="title">
+ 					deliverable.doc
+ 				</ClayCard.Description>
+ 				<ClayCard.Description type="subtitle">
+ 					Stevie Ray Vaughn
+ 				</ClayCard.Description>
+ 				<ClayCard.Detail>
+ 					<ClayLabel displayType="success">{'Approved'}</ClayLabel>
+ 				</ClayCard.Detail>
+ 			</section>
+ 		</div>
+ 	</ClayCard.Body>
+ </ClayCard>
```

To get the behavior of having a ClayCardGrid with Cards, use the following composition with ClayCard:

```diff
- <ClayCardGrid spritemap={spritemap} items={items} schema={schema} />
+ <ClayCard.Group label="Test Files">
+	<CardFile spritemap={imageOrSpritemap} />
+	<CardFile spritemap={imageOrSpritemap} />
+	<CardFile spritemap={imageOrSpritemap} />
+ </ClayCard.Group>
+ <ClayCard.Group label="Test Users">
+	<CardUser spritemap={imageOrSpritemap} />
+	<CardUser spritemap={imageOrSpritemap} />
+	<CardUser spritemap={imageOrSpritemap} />
+ </ClayCard.Group>
```

## ClayCardGrid

> ClayCardGrid was deprecated due ClayCard.Group.

Reviewer: ```suggestion
ClayCardGrid was deprecated due to ClayCard.Group.
```
